models/CourseInitialData.js

```diff
@@ -0,0 +1,65 @@
+courseData = [
```

Comment:
JSON file would be a better option here.

Reply:
Done